deltatech_product_tracking/models/product.py

```diff
@@ -17,61 +30,63 @@ def onchange_tracking(self):
     def write(self, vals):
```

Comment:
la fel si aici:
    def write(self, vals):
        products_without_lot = self.filtered(lambda p: p.tracking == "none")
        res = super(ProductProduct, self).write(vals)
        if products_without_lot and vals.get("tracking", "none") == "lot":
            products_without_lot.fill_lot_in_stock_docs()
        return res